src/pam_radius_auth.h

```diff
@@ -58,6 +58,7 @@
 #define MAXPROMPT 33               /* max prompt length, including '\0' */
 #define DEFAULT_PROMPT "Password"  /* default prompt, without the ': '  */
 
+#define MAXHOSTNAME 65
```

Comment:
Because of
```
/usr/include/x86_64-linux-gnu/sys/param.h:# define MAXHOSTNAMELEN	HOST_NAME_MAX
/usr/include/x86_64-linux-gnu/bits/local_lim.h:#define HOST_NAME_MAX		64
```
so
```
gethostname(hostname, sizeof(hostname) - 1);
```
in order to enforce a null terminated string.

It has been aligned into the last update.